qutebrowser/browser/commands.py

```diff
@@ -2171,6 +2171,12 @@ def fullscreen(self, leave=False):
 
         window = self._tabbed_browser.window()
         if window.isFullScreen():
-            window.showNormal()
+            window.setWindowState(window.last_non_fullscreen_window_state)
+            window.last_window_state = window.last_non_fullscreen_window_state
+            self.jseval('document.webkitExitFullscreen()', quiet=True)
```

Comment:
I'm not convinced we should exit the page fullscreen when exiting fullscreen, for various reasons:

- When going out of fullscreen via the WM, this wouldn't apply, which is inconsistent
- It can be useful to have the window windowed, but the video still full-window in it

(and if we keep this, it should use `tab.action.exit_fullscreen()` like above)